src/padplus-manager/padplus-manager.ts

```diff
@@ -138,7 +138,7 @@ export class PadplusManager extends EventEmitter {
     this.getRoomMemberQueue = new DelayQueueExecutor(500)
     this.resetThrottleQueue = new ThrottleQueue<string>(5000)
     this.resetThrottleQueue.subscribe(async reason => {
-      log.silly(PRE, 'resetThrottleQueue.subscribe() reason: %s', reason)
+      log.info(PRE, 'resetThrottleQueue.subscribe() reason: %s', reason)
```

Comment:
If we are logging this as `info` level, we should make sure the log states the situation clearly.

Reply:
Sure, this log will only be shown when try to reconnect.